cmd/sweepaccount/main.go

```diff
@@ -196,7 +196,9 @@ func makeDestinationScriptSource(rpcClient *rpcclient.Client, accountName string
 	// GetNewAddress always returns a P2PKH address since it assumes
 	// BIP-0044.
 	newChangeScript := func() ([]byte, error) {
-		destinationAddress, err := rpcClient.GetNewAddress(accountName)
+		destinationAddress, err := rpcClient.GetNewAddress(
+			accountName, "legacy",
+		)
```

Comment:
You'll be able to revert this when you move to btcd/v0.23.1.

I also see you have the `sweepaccount` binary accidentally checked in.